lib/models/device.coffee

```diff
@@ -51,6 +51,10 @@ getDeviceModel = (deps, opts) ->
 
 	exports = {}
 
+	# Infer dashboardUrl from apiUrl if former is undefined
+	if not dashboardUrl?
+		dashboardUrl = apiUrl.replace(/api/, 'dashboard')
```

Comment:
Ok so `apiUrl` is set as a default options if not passed to the SDK factory, so I guess we are fine here: https://github.com/resin-io/resin-sdk/blob/master/lib/resin.coffee#L60-L65